libsql/internal/http/http.go

```diff
@@ -3,61 +3,90 @@ package http
 import (
 	"bytes"
 	"context"
+	"database/sql/driver"
```

Comment:
This does not belong here. All dependencies on database/sql should live in the http/driver.go